lib/HTTP/Response.pm

```diff
@@ -30,6 +30,8 @@ sub parse
 	$str = "";
     }
 
+    $status_line =~ s<\x0d\z><>;
```

Comment:
The parent HTTP::Message does `s/\r\z//`.
https://github.com/libwww-perl/HTTP-Message/blob/b7c38e8230663214dd7f8886cebaec92363f8405/lib/HTTP/Message.pm#L66
I think it is better if HTTP::Response follows that.

Reply:
Sounds reasonable, but are \r and \n always 0x0d and 0x0a?

I thought Windows reversed them, but just now I tried it out on Strawberry, and they seem to be the same octets as in *nix.

Anyway, I’ll update it.